tests/parm/control_ca.nml.IN

```diff
@@ -3,11 +3,16 @@
   chksum_debug = .false.
   dycore_only = .false.
   fdiag = @[FDIAG]
+  fhmax = 384
```

Comment:
May I ask why those lines are added? Those values are the default values in the model code.

Reply:
I don't know why that file was changed; some of these changes did not come from me. I'll revert that file and see if the regression tests pass.

Reply:
Those four lines have been removed.

The later lines, to add `ldiag3d=@[LDIAG3D]`, etc. are intentional, and will be set to the model defaults if diagnostics are disabled, or turned on if diagnostics are enabled.